token-service/src/main/java/org/p2p/token/service/manager/TokenServiceEvent.kt

```diff
@@ -0,0 +1,12 @@
+package org.p2p.token.service.manager
+
+import org.p2p.token.service.model.TokenServiceMetadata
+import org.p2p.token.service.model.TokenServicePrice
+
+sealed interface TokenServiceEvent {
+    object Loading : TokenServiceEvent
+    data class TokensPrice(val result: Map<String,TokenServicePrice>) : TokenServiceEvent
```

Comment:
doesn't sound like an event for me imho
TokensPrice  -> TokensPriceLoaded

also I would love to have a documentation for Idle class